### PR TITLE
Add DILI from TDC

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

**Review by phalem — COMMENTED** [review]
remove compound id

File: data/Drug_Induced_Liver_Injury/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  \ withdrawals for the past 50 years (e.g. iproniazid, ticrynafen, benoxaprofen).\
+  \ This dataset is aggregated from U.S. FDA\u2019s National Center for Toxicological\
+  \ Research."
+targets:
+- id: mutagenic
+  description: whether it can cause liver injury (1) or not (0).
+  units: liver_injury
+  type: categorical
+  names:
+  - DILI
```
Comment (kjappelbaum) [inline data/Drug_Induced_Liver_Injury/meta.yaml:13]:
```suggestion
```

not sure if we can expect this acronym to be learned by model

**Review by phalem — COMMENTED** [review]
Add benchmark field

**Comment by phalem** [comment]
Great! Thanks @MicPie  for helping and adding missing field. I will try to solve other data as well.

File: data/drug_induced_liver_injury/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+targets:
+    - id: liver_injury
+      description: whether it can cause liver injury (1) or not (0).
+      units:
+      type: boolean
+      names:
+          - liver injury
+          - drug induced liver injury
+          - fatal liver disease caused by drugs
+          - liver toxicity
```
Comment (kjappelbaum) [inline data/drug_induced_liver_injury/meta.yaml:18]: @MicPie @phalem 

While we can ask an LLM to rephrase our prompt, I think it'd be great if we could already now try to use consistent declination/phrasing. For instance, names that can easily be inserted in `Is <name> <identifier>`  (`or what molecules in the list are <name>?` In the case of `boolean` targets (see contribution guide).

File: data/drug_induced_liver_injury/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      uris:
+          - http://purl.bioontology.org/ontology/MEDDRA/10072268
+          - http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C84427
```
Comment (kjappelbaum) [inline data/drug_induced_liver_injury/meta.yaml:21]:
```suggestion
```
didn't resolve?

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
Only small comments about the target names, otherwise nice!

File: data/drug_induced_liver_injury/meta.yaml (hunk 2 above)
Comment (MicPie) [reply data/drug_induced_liver_injury/meta.yaml:18]: @kjappelbaum
I'm not sure what is better:
* Adding (`What molecules in the list are`) `causing a liver injury`, `causing a liver toxicity` or simply add `causing a` to the prompt.
* Or we change everything to (`What molecules in the list are`) `injuring the liver`,`toxic to the liver`, etc.

**Pull request merged**